python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -1290,13 +1290,13 @@ def _ensure_persisted_job_snapshot(
         job_snapshot: "JobSnap",
         parent_job_snapshot: "Optional[JobSnap]",
     ) -> str:
-        from dagster._core.snap import JobSnap, create_job_snapshot_id
```

Comment:
i think you should just search and replace all callsites to `create_job_snapshot_id` and kill it

Reply:
done